### PR TITLE
Added solidly to dex.trades

### PR DESCRIPTION
**Pull request by sophie-analytics**
# Thank you for contributing to Spellbook!

Thank you for taking the time to submit code in Spellbook. A few things to consider:

- If you are a first-time contributor, please sign the CLA by copy & pasting **_exactly_** what the bot mentions in PR comment
- Refer to [docs](#spellbook-contribution-docs) section below to answer questions
- Dune team will review submitted PRs as soon as possible

## Spellbook sub-projects

To prepare for Spellbook to scale in size, Dune team has started to pilot sub-projects:

- For a quick overview, see [the GH discussion on the topic](https://github.com/duneanalytics/spellbook/discussions/5238)
  - Please ask questions and continue the conversation there
- The first spell lineage to move: `tokens.erc20` and all upstream spells which help build it
  - **_Note_**: Please navigate to [tokens](/tokens) root directory to find tokens objects

## Spellbook contribution docs

The [docs](docs) directory has been implemented to answer as many questions as possible. Please take the time to reference each `.md` file within this directory to understand how to efficiently contribute & why the repo is designed as it is 🪄

Example questions to be answered:

- What does each property in the [model config block](/docs/models/model_config_block.md) mean?
- What is the [CI test](/docs/ci_test/ci_test_overview.md) attached to PRs and how can I best utilize it?
- What are the Spellbook [best practices](/docs/general/best_practices.md)?

Please navigate through the [docs](/docs) directory to find as much info as you can.

**Note**: happy to take PRs to improve the docs, let us know 🤝

### CODE REVIEW

File: seeds/_sector/dex/_schema.yml [hunk 1]
```diff
@@ -3001,3 +3001,18 @@ seeds:
         token_bought_amount_raw: uint256
         token_sold_amount_raw: uint256
         block_date: timestamp
+
+  - name: solidly_v3_base_base_trades_seed
```
Comment (Hosuke) [inline seeds/_sector/dex/_schema.yml:3005]: We may rename the seed name as `solidly_base_base_trades_seed`
```suggestion
  - name: solidly_base_base_trades_seed
```

**Review by Hosuke — CHANGES_REQUESTED** [review]

**Review by Hosuke — APPROVED** [review]
LGTM.✅
Thank you @sophie-analytics 

[Sample solidly_v3_base_base_trades](https://dune.com/queries/3484973)

File: sources/_sector/dex/trades/base/_sources.yml [hunk 2]
```diff
@@ -138,3 +138,7 @@ sources:
     tables:
       - name: GridFactory_evt_GridCreated
       - name: pool_evt_Swap
+  - name: solidly__base
```
Comment (jeff-dude) [inline sources/_sector/dex/trades/base/_sources.yml:141]: wonder if this was a bug in decoding, to have two `_` in the schema name?

**Review by jeff-dude — APPROVED** [review]
left a minor comment below. nothing that blocks the code from being merged, but curious for long term if the schema should be cleaned up

**Pull request merged**